website/docs/reference/global-configs.md

```diff
@@ -103,46 +103,55 @@ config:
 
 </File>
 
-## Failing fast
+<VersionBlock firstVersion="1.1">
 
-Supply the `-x` or `--fail-fast` flag to `dbt run` to make dbt exit immediately if a single resource fails to build. If other models are in-progress when the first model fails, then dbt will terminate the connections for these still-running models. 
+### Cache database objects for selected resource
 
-For example, you can select four models to run, but if a failure occurs in the first model, the failure will prevent other models from running:
+:::caution Experimental config flag
+This should be considered an _unstable_ config flag because it is experimental and subject to change. We reserve the right to make breaking changes to this config flag.
+:::
+
+At the start of runs, dbt caches metadata about all the objects in all the schemas where it might materialize resources (such as models). By default, dbt caches all schemas related to the project. When this config is enabled, dbt will only cache schemas related to selected resources for the current run. This can offer significant speed improvements when running a small subset of a large project.
+
+For example, to improve speed and performance while focused on Salesforce models you would select the source and use the `cache-selected-only` flag:
```

Comment:
This feels to me like an important nuance to capture, even if it makes the sentence clunky:
```suggestion
For example, to improve speed and performance while focused on developing Salesforce models, which are materialized into their own dedicated schema, you would select those models and pass the `cache-selected-only` flag:
```